include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -29,18 +29,18 @@ template <typename _KernelNameBase, uint16_t __wg_size = 256 /*work group size*/
           uint16_t __req_sub_group_size = (__block_size < 4 ? 32 : 16)>
 struct __subgroup_radix_sort
```

Comment:
Comment for the line 20. Sorry, but this is a hack. Unless we have strong reason to do that, I believe we should not make our code so implicit and unreadable. Every header should be self-contained. Here I don't see any include directive. Let's fix that.

Also the include guard is not consistent with everything else.